templates/android/library/src/main/java/io/package/extensions/TypeExtensions.kt.twig

```diff
@@ -1,9 +1,76 @@
 package {{ sdk.namespace | caseDot }}.extensions
 
+import {{ sdk.namespace | caseDot }}.models.Payload
 import kotlin.reflect.KClass
 import kotlin.reflect.typeOf
 
 inline fun <reified T : Any> classOf(): Class<T> {
     @Suppress("UNCHECKED_CAST")
     return (typeOf<T>().classifier!! as KClass<T>).java
-}
\ No newline at end of file
+}
+
+
+fun String.fromMultiPart(): Map<String, Any> {
```

Comment:
instead of `fromMultiPart`, `parseMultiPart` might make more sense, as we are not creating string from multi part, this function is parsing string as a multi part object?

Reply:
It looks like in all other runtimes the function named `parseMultiPart` as well